### PR TITLE
Adding 3rd layout option and preloading images [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- Adding 16x9 layout selection in settings
- Adding 16x9 layout HTML
- Preloading all images via use of Images module and ensuring to preload images again upon a data refresh
- Applying specific image dimensions when layout is 16x9
- Adding unit test coverage for Images module
- Revising integration tests and using a mock for Image module to bypass the actual loading of an image

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks. Note, I'm using a mock of the Image module for all integration tests. It's an exact replica with the only difference being it skips the actual load - https://goo.gl/qniDjY. This is so our integration tests don't need tricky and convoluted workarounds in order to wait for the images to actually get loaded before the tests get run. I don't think it would add any benefit to our tests, hence the use of the mock. 

Presentation - http://rva.risevision.com/#/PRESENTATION_MANAGE/id=6b1e3109-aae9-438a-ac62-82246bf29d47?cid=30007b45-3df0-4c7b-9f7f-7d8ce6443013

File: src/widget/content.js [hunk 1]
```diff
@@ -154,14 +165,39 @@ RiseVision.RSS.Content = function (prefs, params) {
...
 
-  function _getTemplate(item) {
+  function _getImageDimensions(image) {
+    var dimensions = null,
+      ratioX, ratioY, scale;
+
+    switch (params.layout) {
+      case "layout-16x9":
+        dimensions = {};
+        dimensions.width = prefs.getString("rsW") - 20; // 20px padding left & right
```
Comment (donnapep) [inline src/widget/content.js]: @stulees This is fairly brittle. Can we get the padding on the image element using Javascript instead?

**Comment by stulees** [comment]
@donnapep In order to get the padding values I've had to set the image dimensions after the item has been appended to the DOM. Please review last commit, presentation is the same. Thanks

**Comment by donnapep** [comment]
@stulees Looks good. :+1:

**Pull request merged**